dependencies.list

```diff
@@ -1,4 +1,4 @@
 PACKAGE_NAME=realm-core
 VERSION=11.0.0-beta.1
 OPENSSL_VERSION=1.1.1g
-MDBREALM_TEST_SERVER_TAG=2020-12-02
+MDBREALM_TEST_SERVER_TAG=2021-03-09
```

Comment:
There was some breaking changes in the format of the docker image and you are going to need the CI changes from https://github.com/realm/realm-core/pull/4423 once that is finalized and merged.